content/v3/enterprise/management_console.md

```diff
@@ -47,6 +47,8 @@ When using this endpoint, your Enterprise instance must have a password set. Thi
 1. If you're working directly with the API before accessing the web interface, you must pass in the password parameter to set your password.
 2. If you set up your instance via the web interface before accessing the API, your calls to this endpoint do not need the password parameter.
 
+You may also need to add the `-k` flag when using `curl`, since GitHub Enterprise uses a self-signed certificate before you [add your own SSL certificate](https://help.github.com/enterprise/admin/guides/installation/dns-hostname-subdomain-isolation-and-ssl/#ssl).
```

Comment:
@mellybess I would add a small bit about what the `-k` flag does or at least add a link to http://curl.haxx.se/docs/manpage.html#-k (not sure if linking out to external pages in these docs is cool or not).

Also since this tip applies when using any self-signed certificate it could apply to all endpoints. I'm wondering if it would make sense to move it up towards the top of the document after the tip section.